concourse/steps/component_descriptor.py

```diff
@@ -81,7 +82,7 @@ def base_component_descriptor_v2(
         ],
         componentReferences=[], # added later
         resources=[], # added later
-        labels=[], # added later
+        labels=component_labels,
```

Comment:
if this attr is to be modified later, defaulting it to a tuple is rather unhandy (I already changed this to a list, though)..